packages/wrangler/src/dev/inspect.ts

```diff
@@ -252,19 +253,25 @@ export default function useInspector(props: InspectorProps) {
 				timeout: 2000,
 				abortSignal: abortController.signal,
 			});
-			server.listen(props.port);
+			server.listen(props.port, "127.0.0.1");
 		}
 		startInspectorProxy().catch((err) => {
 			if ((err as { code: string }).code !== "ABORT_ERR") {
 				logger.error("Failed to start inspector:", err);
 			}
 		});
 		return () => {
-			server.close();
+			//server.close();
```

Comment:
Should this be completely removed if you replace it with the graceful terminator below?